language/moe/SUT.py

```diff
@@ -0,0 +1,367 @@
+import os
+import time
+import numpy as np
+import array
+import torch
+from torch.nn.functional import pad
+from torch.utils.data import DataLoader
+from transformers import AutoModelForCausalLM, AutoTokenizer
+from transformers.generation.streamers import BaseStreamer
+
+import pickle
+import time
+import threading
+import tqdm
+import queue
+
+import logging
+from typing import TYPE_CHECKING, Optional, List
+from pathlib import Path
+
+import mlperf_loadgen as lg
+from dataset import Dataset
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger("Mixtral-8x7B-v0.1")
+
+gen_kwargs = {
+    "early_stopping": True,
+    "max_new_tokens": 1024,
+    "min_new_tokens": 1,
+    "num_beams": 1,
+    "do_sample": False
+}
+
```

Comment:
Can you help run autopep8 on the python file? The code style is a bit weird

Reply:
15000 should be ok

Reply:
Done